replicated_kv_store/raft_node.go

```diff
@@ -39,13 +40,17 @@ type RaftNode struct {
 	kvstore_addr          string                          // stores respective port on which local key value store is running
 	commits_ready         chan int32                      // Channel to signal the number of items commited once commit has been made to the log.
 	commits_applied_to_kv chan bool                       // Channel to indicate completion of changes applied to key value store
+
+	trackMessage map[string][]string // tracks messages sent by clients
```

Comment:
If we assume that the client doesn't perform another request before getting an ack for its previous request, will a single string as the value in the map be sufficient?

Reply:
Yeah bro then it should be fine.